content/swagger/resources/app_catalogs/routes/app_catalogs_id_mapped_products.yaml

```diff
@@ -1,13 +1,16 @@
 get:
-  summary: Get the list of mapped products related to a catalog
+  summary: Get the list of mapped fattened products or simple products related to a catalog
```

Comment:
```suggestion
  summary: Get the list of mapped flattened products or simple products related to a catalog
```

Reply:
On retourne tjs des flattened products, ou product that correspond to the catalog mapping ?